pyscf/cc/__init__.py

```diff
@@ -183,29 +183,27 @@ def RQCISD(mf, frozen=None, mo_coeff=None, mo_occ=None):
 RQCISD.__doc__ = qcisd.QCISD.__doc__
 
 
-def FNOCCSD(mf, thresh=1e-6, pct_occ=None, nvir_act=None, frozen=None):
+def FNOCCSD(mf, thresh=1e-6, pct_occ=None, pvir_act=None, nvir_act=None, frozen=None):
     """Frozen natural orbital CCSD
 
     Attributes:
         thresh : float
             Threshold on NO occupation numbers. Default is 1e-6 (very conservative).
         pct_occ : float
             Percentage of total occupation number. Default is None. If present, overrides `thresh`.
+        pvir_act : float
+            Percentage of total number of virtuals (ceiling is taken to get an integer number).
+            Default is None. If present, overrides `thresh` and `pct_occ`.
         nvir_act : int
             Number of virtual NOs to keep. Default is None. If present, overrides `thresh` and `pct_occ`.
     """
     import numpy
-    if frozen is None:
-        frozenocc = 0
-    else:
-        assert(isinstance(frozen, (int,numpy.int64)))
-        frozenocc = frozen
     from pyscf import mp
-    pt = mp.MP2(mf).set(verbose=0).run()
-    frozen, no_coeff = pt.make_fno(thresh=thresh, pct_occ=pct_occ, nvir_act=nvir_act)
-    frozen = numpy.hstack([numpy.arange(frozenocc),frozen])
```

Comment:
Should there be a comment in the docstring how `frozen` passed by the user is being used?